main.js

```diff
@@ -0,0 +1,305 @@
+  renderPageForecast();
+};
+//renders current weather onto page
+var renderPage = function () {
+  $('.weather').empty();
+  var source = $('#weather-template').html();
+  var template = Handlebars.compile(source);
+  var newHTML = template(weatherToday[0]);
+ $('.weather').append(newHTML);
+};
+//renders forecast onto page
+var renderPageForecast = function () {
+  $('.weather-two').empty();
+  var source = $('#forecast-template').html();
+  var template = Handlebars.compile(source);
+  var newHTML = template(forecast[0]);
+ $('.weather-two').append(newHTML);
+};
+//takes forecast data and splits into array for each day of week
+var forecastDataToArrays = function (data) {
+  for (i = 0; i < 8; i++) {
+    daysOfWeekData[0].push(data.list[i]);
+  }
+  for (i = 8; i < 16; i++) {
+    daysOfWeekData[1].push(data.list[i]);
+  }
+  for (i = 16; i < 24; i++) {
+    daysOfWeekData[2].push(data.list[i]);
+  }
+  for (i = 24; i < 32; i++) {
+    daysOfWeekData[3].push(data.list[i]);
+  }
+  for (i = 32; i < 40; i++) {
+    daysOfWeekData[4].push(data.list[i]);
+  }
+};
+//iterates through data for each day of the week to get an average temp
+var getEachDayTemp = function () {
+  for (i = 0; i < daysOfWeekData.length; i++) {
+    var averageTemp = 0;
+    var tempComputed = 0;
+    for (j = 0; j < daysOfWeekData[i].length; j++) {
+      averageTemp += daysOfWeekData[i][j].main.temp;
+    };
+    tempComputed = (Math.round(averageTemp / 8)) + "\xB0";
+    forecastDataComputed[i][0] = tempComputed;
+  };
+  averageTemp = 0;
+};
+//uses the timestmap for the first day of the week to get the weekday as an index number, takes that number and checks it against dayOfWeek array for string value to display for each day
+var getDayOfWeek = function () {
+  var firstDayTimeStamp = daysOfWeekData[0][0].dt;
+  var xx = new Date();
+  xx.setTime(firstDayTimeStamp*1000);
+  theDay = xx.getDay();
+};
+//gets the condition for each day of forecast, reflects forecast for current time on each day
+var getEachDayCondition = function () {
+  var dailyCondition = "";
+  for (i = 0; i < daysOfWeekData.length; i++) {
+    dailyCondition = daysOfWeekData[i][0].weather[0].main;
+    forecastDataComputed[i].push(dailyCondition);
+  };
+  dailyCondition = "";
+}
+//gets the icon number for each day of forecast
+var getEachDayConditionIcon = function () {
+  var dailyConditionURL = "";
+  for (i = 0; i < daysOfWeekData.length; i++) {
+    dailyConditionURL = daysOfWeekData[i][0].weather[0].icon;
+    forecastDataComputed[i].push(dailyConditionURL);
+  };
+  dailyConditionURL = "";
+};
+//checks if a default is set
+var checkIfDefault = function () {
+  if (sessionStorage.hasOwnProperty('userDefault')) {
+    var getDefault = sessionStorage.getItem('userDefault');
+    $('.city-name-input').attr("placeholder", getDefault);
+    fetch(getDefault);
+    fetchForecast(getDefault);
+  };
+};
+//success function for auto location lookup
+function success(pos) {
```

Comment:
I'm not seeing where this is being called. Also, good comments, but the function var name itself could use something more descriptive (than just success)